code/web/src/modules/user/MyProducts.js

```diff
@@ -0,0 +1,155 @@
+//Imports
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Link } from 'react-router-dom'
+import { Helmet } from 'react-helmet'
+
+
+// UI Imports
+import { Grid, GridCell } from '../../ui/grid'
+import { H3, H4 } from '../../ui/typography'
+import Button from '../../ui/button'
+import { grey, grey2, black } from '../../ui/common/colors'
+
+//App Imports
+import SubNav from './SubNav'
+import userRoutes from '../../setup/routes/user'
+import { logout } from './api/actions'
+
+class MyProducts extends PureComponent {
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      viewAll: true
+    }
+  }
+
+  toggleKept = () => {
+    // event.preventDefault()
```

Comment:
If we're not passing in event then we can lose that line or refactor to use it, seems to function okay.